src/query/query.pest

```diff
@@ -10,17 +10,24 @@ expression         =  { expr }
 funcNames          = _{ latest }
 latest             =  { "latest" }
 
-infixExpression = { firstArg ~ infixFunction ~ secondArg }
+infixExpression = { firstArg ~ infixFunction ~ lookupValue }
```

Comment:
Changing the name here because then we get slightly better errors

Reply:
Would it also be worth renaming `firstArg` to `lookup` or `lookupLhs` for the same reason?

Reply:
Yes, can actually remove `firstArg` as a thing altogether. Tidied this up, and in the corresponding parsing code